code/modules/mob/interactive.dm

```diff
@@ -566,7 +566,7 @@
 	if((TARGET && (doing & FIGHTING)) || graytide) // this is a redundancy check
 		var/mob/living/M = TARGET
 		if(istype(M,/mob/living))
-			if(M in range(FUZZY_CHANCE_LOW,src))
+			if(M in range(MIN_RANGE_FIND,src))

```

Comment:
On a side note: I wish we had a proper mobrange proc (like orange and
trange)
Op 23-mrt.-2015 15:14 schreef "Alex" <notifications@github.com>:

> In code/modules/mob/interactive.dm
> <https://github.com/tgstation/-tg-station/pull/8467#discussion_r26939144>:
>
> > @@ -566,7 +566,7 @@
> >  	if((TARGET && (doing & FIGHTING)) || graytide) // this is a redundancy check
> >  		var/mob/living/M = TARGET
> >  		if(istype(M,/mob/living))
> > -			if(M in range(FUZZY_CHANCE_LOW,src))
> > +			if(M in range(MIN_RANGE_FIND,src))
>
> There's room for improvement here, range() is an expensive proc because it
> produces a list of atoms which is located all around the given object (src
> in this case).
>
> If you are simply checking if the mob is in range then I think the best
> solution would be to replace it with get_dist(get_turf(M), get_turf(src)).
> This operation will take the X, Y coordinates of both turfs and will return
> the distance between them, which is MUCH more efficient to use.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tgstation/-tg-station/pull/8467/files#r26939144>.
>